rl/memory.py

```diff
@@ -59,6 +59,39 @@ def append(self, v):
             raise RuntimeError()
         self.data[(self.start + self.length - 1) % self.maxlen] = v
 
+class FixedBuffer(object):
```

Comment:
In light of the findings in #165, maybe we could replace it with something simpler?